### PR TITLE
Console feature to temporarily add ICRC-1 token

### PR DESCRIPTION
**Pull request by dskloetd**
# Motivation

Sometimes people accidentally send tokens to their NNS dapp address while those tokens are not supported by NNS dapp.
It would be nice if there was a way to recover those tokens.

The experience could definitely be improved but I don't want to spend more time on this now, so please review based on whether this is an improvement or not.

# Changes

1. Provide a hard coded logo for unknown tokens.
2. Add a method to the `window` object to add a ledger and index canister to the store.

# Tests

1. Adjusted an existing unit test for an unknown token.
2. Tested manually adding an unknown ledger.

Currently it happens to work with and without index canister but there are no guarantees:
```
__experimentalAddIcrc1Token('m7h5t-euaaa-aaaaa-qabia-cai');
__experimentalAddIcrc1Token('m7h5t-euaaa-aaaaa-qabia-cai', 'myg3h-jmaaa-aaaaa-qabiq-cai');
```
image

# Todos

- [x] Add entry to changelog (if necessary).

### CODE REVIEW

File: frontend/src/lib/stores/icrc-canisters.store.ts [hunk 1]
```diff
@@ -54,3 +56,30 @@ const initIcrcCanistersStore = (): IcrcCanistersStore => {
 };
 
 export const icrcCanistersStore = initIcrcCanistersStore();
+
+// Useful to help people record coins they accidentally sent to NNS dapp, but
+// which are not officially supported by the NNS dapp.
+//
+// In the devtools console, run:
+// __experimentalAddIcrc1Token(ledgerCanisterId, indexCanisterId)
+if (browser) {
```
Comment (mstrasinskis) [inline frontend/src/lib/stores/icrc-canisters.store.ts:65]: Will using the `triggerDebugReport` improve the ux a bit?

Comment (dskloetd) [reply frontend/src/lib/stores/icrc-canisters.store.ts:65]: Also, we should really just implement this properly for everybody.
This is just to be able to support for lost coins. I don't care about the UX tbh.

Comment (mstrasinskis) [reply frontend/src/lib/stores/icrc-canisters.store.ts:65]: No objections :)

**Review by mstrasinskis — APPROVED** [review]
The code looks good to me. Thanks!

**Review by lmuntaner — APPROVED** [review]
LGTM

**Pull request merged**